tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
We are getting there, but let's remove this file anyway :-)

1) Of course it is shorter command. But then we should make here scripts build_and_test_on_cpu.sh, build_and_test_on_gpu.sh, build_and_test_on_cpu_py2x.sh, build_and_test_on_cpu_py3x.sh, etc.
2) it is not test on install, it is "build, install and test pip inside docker".sh

What if you want to run without docker? You can't simply drop the docker script. All the other builds work just fine without the "tensorflow/tools/ci_build/ci_build.sh CONTAINER_TYPE" thing at the beginning. And that is the same command live we run on mac. Would this script work on mac where there is no docker?

If anything, what we could do would be:
1) split the current gpu.sh
  - the only thing that would stay would be build, install and call test_installation.sh
  - note: that if there is such a feature as SKIP_TESTS feature would be there.
2) create test_installation.sh
  - this one would be the majority of the current script
  - it would run the python tests without blaze, java, and other things
  - such a script could be used to validate docker container we distribute as well as maybe one day some deb and rpm packages
3) and we would still not need this script ;-)